style.css

```diff
@@ -1,63 +1,76 @@
 body {
-  max-width: 375px;
+  min-width: 375px;
   font-family: 'Poppins', sans-serif;
   background-color: #e5e5e5;
   margin: 0;
 }
 
-header {
-  max-width: 330px;
-  margin: 7.5px auto;
+.header-section {
+  margin: 2% 6%;
   display: flex;
   align-items: center;
   justify-content: space-between;
   padding: 0 22.5px;
+  width: auto;
+  position: relative;
 }
 
 .logo {
-  height: 20px;
-  width: 76px;
+  height: 40px;
+  width: 120px;
   text-decoration: none;
-  line-height: 20px;
-  font-size: 20px;
+  line-height: 40px;
+  font-size: 24px;
   font-weight: 700;
   font-style: normal;
   white-space: nowrap;
-  top: 50px;
+}
+
+#projects-desktop {
+  display: none;
 }
 
 #introduction-paragraphs {
-  margin-right: 2%;
-  padding: 25px 0 22.9% 0;
-  flex-direction: column;
-  display: flex;
+  padding: 0%;
+  margin: 0%;
+  border-bottom-left-radius: 75px;
+  padding-top: 3%;
+}
+
+#nav-menu {
+  display: none;
+}
+
+.desktop-image-div {
+  display: none;
 }
 
 #main-div {
+  max-width: 1024px;
   padding: 0%;
-  margin: 0%;
+  margin: 0 auto;
   background-image: url(./images/background/bg.png);
   background-repeat: no-repeat;
   background-position-x: 0%;
   background-position-y: 90%;
   border-bottom-left-radius: 75px;
-  padding-top: 2.9%;
+  padding-top: 50px;
   background-color: #fff;
 }
```

Comment:
- [ ] The max-width is making your headline section smaller than the screen on any screen bigger than 1024px. It is advised not to use fixed widths for responsiveness. Kindly consider using a width of 100%. You can restrict the width of contents using padding with a value in percentage if you want to.
- [ ] Kindly reduce the spacing above the header to match the design template. You are expected to stick to design as much as possible, this includes margins and paddings.
### Figma
![image](https://user-images.githubusercontent.com/94551484/229279859-1118f610-94d0-4b16-bb2a-ed7ed737cac9.png)

### Your design
![image](https://user-images.githubusercontent.com/94551484/229279835-40b28ae9-ed42-4919-8a33-7c122c5bc945.png)